tests/src/test/scala/cats/tests/MonoidTests.scala

```diff
@@ -1,22 +1,29 @@
 package cats
 package tests
 
-import org.scalatest._
-
 import cats.functor._
 
-class MonoidTests extends FunSuite {
+class MonoidTests extends CatsSuite {
   {
-    import cats.implicits._
```

Comment:
Should we maybe include a comment specifying that the instances below should be found when all the instances are in scope?
It might not be as obvious what we are doing here, now that we moved to `CatsSuite`.